invokeai/app/api/dependencies.py

```diff
@@ -45,6 +45,7 @@ class ApiDependencies:
 
     @staticmethod
     def initialize(config, event_handler_id: int):
+        TODO: Get rid of Globals
         Globals.try_patchmatch = config.patchmatch
```

Comment:
Should we fold `Globals` entirely into `InvokeAIAppConfig` and make every option a CLI argument?

```
$ invokeai-node-web
...
File "/home/clipclop/Code/invokeai/invokeai/app/api/dependencies.py", line 49, in initialize
    Globals.try_patchmatch = config.patchmatch
AttributeError: 'InvokeAIWebConfig' object has no attribute 'patchmatch'
```

Reply:
I'm gradually getting rid of Globals. This code is now removed.